install-panel.sh

```diff
@@ -589,34 +638,18 @@ function centos8_dep {
 ## OTHER OS SPECIFIC FUNCTIONS ##
 #################################
 
-function ubuntu_universedep {
```

Comment:
Thank you for removing this. Why did I even implement it like this in the first place? Makes no sense even to me :slightly_smiling_face:

Reply:
Hehe....